build.cmd

```diff
@@ -20,16 +20,13 @@ powershell -NoProfile -ExecutionPolicy unrestricted -Command ^
 -AzureFeed 'https://dotnetcli.blob.core.windows.net/dotnet' "
 
 :globaljson
-rem User expected to provide global.json with allowPrerelease=true
-if not exist %~dp0global.json (
```

Comment:
Why was there a condition originally?

Reply:
to give devs the flexibility of moving up to a newer .NET SDK before the build did - but it's proved more trouble, as it basically locks in the current version when the build itself moves up